src/services/CreateAnalysisService.tsx

```diff
@@ -6,11 +6,11 @@ import NotificationService from "./notificationService";
 import { formatDate } from "../shared/utils";
 
 export interface StudyInstance {
-  studyInstanceUID: string;
-  studyDescription: string;
-  modality: string;
-  createdDate: string;
-  setModelType?: (modality: string) => void;
+  studyInstanceUID: string,
+  studyDescription: string,
+  modality: string,
+  studyDate: string,
+  setModelType?: (modality: string) => void
```

Comment:
Related comment: https://github.com/darwinai/covidnet_ui/pull/49/files#r616263005
Please, ensure that the related comment is resolved.